packages/assistify-help-request/client/views/creationDialog/AssistifyCreateExpertise.js

```diff
@@ -199,17 +199,17 @@ Template.AssistifyCreateExpertise.onCreated(function() {
 			Meteor.call('roomDisplayNameExists', name, (error, result) => {
 				if (error) {
 					return;
-				}
-				if (result) {
+				} else {
```

Comment:
@vickyokrm I prefer this to be explicit `if...else` compared to returning early in order to prevent subsequent execution. And particularly, the earlier `if (result)` is just wrong for booleans ;)